templates/addons/storageclass-azure-disk.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: storage.k8s.io/v1beta1
+apiVersion: storage.k8s.io/v1
```

Comment:
I also changed storage.k8s.io to v1 since that's what we're using in the azure-file spec and that's what the docs (https://kubernetes.io/docs/concepts/storage/storage-classes/#azure-disk) use, let me know if that's not correct